server/service/src/requisition/request_requisition/generate.rs

```diff
@@ -46,6 +49,8 @@ pub fn generate_requisition_lines(
     store_id: &str,
     requisition_row: &RequisitionRow,
     item_ids: Vec<String>,
+    // Need for plugins since sometimes we set id passed in as parameter in graphql
```

Comment:
Well, the `insert_request_requisition_line` that does this is only used from batch mutation, which is not used by our frontend? Do you know if there is an integration that we need to support here, or should we remove the batch mutation and then wouldn't need to do this?

Reply:
I TAKE IT BACK IM BAD AT SEARCHING IT IS INDEED USED

Reply:
🥇